targets/esp32/main.c

```diff
@@ -51,7 +53,8 @@ static void espruinoTask(void *data) {
   jshInit();     // Initialize the hardware
   jswrap_ESP32_wifi_restore();
   jsvInit();     // Initialize the variables
-  vTaskDelay(1000 / portTICK_PERIOD_MS);
+  // not sure why this delay is needed?
+  vTaskDelay(200 / portTICK_PERIOD_MS);
```

Comment:
@jumjum123 
I'm not sure the intention of the the delays? The one at line 38 does not need to be needed.

I could not remove the one at line 57 - it seems some delay was still required - so made it 200 ms